forge-gui/res/cardsfolder/upcoming/mothlight_processionist.txt

```diff
@@ -0,0 +1,12 @@
+Name:Mothlight Processionist
+ManaCost:1 W
+Types:Creature Human Cleric
+PT:2/2
+S:Mode$ Continuous | Affected$ Card.Enchantment+YouCtrl | AffectedZone$ Stack | AddKeyword$ Convoke | Description$ Enchantment spells you cast have convoke.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Enchantment.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigConjure | TriggerDescription$ Eerie — Whenever an enchantment you control enters and whenever you fully unlock a Room, conjure a card named CARDNAME into your hand. Discard that card at the beginning of the next end step.
```

Comment:
named static

Reply:
I don't know what this means haha. I'm sorry, could you elaborate more?

Reply:
same problem as with Verdant Dread